server/portal/templates/includes/header.html

```diff
@@ -22,8 +22,6 @@
 
 
       {% include "includes/nav_cms.html" with className="navbar-nav" only %}
```

Comment:
Restore unnecessarily deleted new line.

```suggestion
      {% include "includes/nav_cms.html" with className="navbar-nav" only %}

```